onnx_tf/handlers/backend/resize.py

```diff
@@ -7,9 +7,28 @@
 from onnx_tf.handlers.handler import ps_description
 from onnx_tf.common.tf_helper import tf_shape
 
+
 @onnx_op("Resize")
 @partial_support(True)
-@ps_description("Resize required 4D input in Tensorflow.")
+@ps_description(
+    "Resize required 4D input in Tensorflow. " +
+    "For opset 11, only the following attributes and inputs conbination " +
+    "are supported in Tensorflow: 1. mode=nearest, " +
+    "coordinate_transformation_mode=asymmetric, nearest_mode=floor, " +
+    "can use scales(*) or sizes. 2. mode=nearest, " +
+    "coordinate_transformation_mode=tf_half_pixel_for_nn, " +
+    "nearest_mode=floor, can use scales(*) or sizes. 3. mode=linear, " +
+    "coordinate_transformation_mode=half_pixel, can use scales(*) or sizes. " +
+    "4. mode=cubic, coordinate_transformation_mode=half_pixel, " +
+    "cubic_coeff_a=-0.5, exclude_outside=1, can use scales(*) or sizes. " +
+    "5. mode=nearest, coordinate_transformation_mode=tf_crop_and_resize, " +
+    "extrapolation_value=any_float_value, nearest_mode=round_prefer_ceil, " +
+    "can use scales or sizes. 6. mode=linear, " +
+    "coordinate_transformation_mode=tf_crop_and_resize, " +
+    "extrapolation_value=any_float_value, can use scales or sizes." +
+    "\nNote (*): The accuracy of your model will go down, if the height and " +
+    "the width of the new sizes(scales * origial sizes) are not in whole numbers."
```

Comment:
add newlines and indent spacing for the numbered bullets and final note so the `support_status.md` notes are more readable

Reply:
@pluradj that's a good idea, just updated the code, please review again.